Gosu/Audio.hpp

```diff
@@ -33,33 +38,38 @@ namespace Gosu
         //! Calling this twice, or too late, does not do any harm.
         void stop();
 
+        //! Returns the current volume of the song.
+        double volume() const;
         //! \param volume Can be anything from 0.0 (silence) to 1.0 (full volume).
         void set_volume(double volume);
+
         //! \param pan Can be anything from -1.0 (left) to 1.0 (right).
         void set_pan(double pan);
         //! \param speed Use 1.0 for normal playback speed.
         void set_speed(double speed);
     };
-    
-    //! A sample is a short sound that is completely loaded in memory, can be
-    //! played multiple times at once and offers very flexible playback
-    //! parameters. Use samples for everything that's not music.
-    class Sample
-    {
-        struct SampleData;
-        std::shared_ptr<SampleData> data;
 
+
+    //! A sound handles all your audio needs from short samples (e.g. explosions) to long songs (e.g. background music)
+    //!
+    //! Use streaming = false for everything that's not music. (Maybe add a more detailed explanation why this is better,
+    //! because some the disadvantages of song might dissappear with the merge?)
+    class Sound
+    {
+        struct SoundData;
+        std::shared_ptr<SoundData> data;
+        std::vector<Channel> current_songs;
```

Comment:
Not sure about the naming but thats basically the global variable you mentioned in #117

Reply:
`streaming_sounds`? This doesn't have to be part of `Audio.hpp`, it can be a static variable in `Audio.cpp`.